Registry/gem_autoload.materialeditor.setreg

```diff
@@ -1,6 +1,9 @@
 {
     "Amazon": {
         "Gems": {
+            "AtomMaterialEditor": {
+                "AutoLoad": false
+            },
```

Comment:
It turns out that the standalone material editor executable tries to load all the gems, but since the new AtomMaterialEditor target (which is a module loaded by the editor) it's registering the same components it fails running the standalone application indicating that we're trying to load the same component twice.

These lines avoid the standalone exe to load the AtomMaterialEditor module.